charts/prow/templates/lego_deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: '{{ template "fullname" . }}.lego'
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        # Required for the auto-create kube-lego-nginx service to work.
+        app: kube-lego
+    spec:
+      containers:
+      - name: kube-lego
+        image: {{ .Values.images.lego }}
+        imagePullPolicy: {{ .Values.images.pullPolicy }}
+        ports:
+        - containerPort: 8080
+        env:
+        - name: LEGO_EMAIL
+          value: "{{ .Values.lego.email }}"
+        - name: LEGO_URL
+          value: "https://acme-v01.api.letsencrypt.org/directory"
```

Comment:
You've created a ConfigMap to contain both the LEGO_EMAIL and LEGO_URL values, but not actually used them in the Deployment. Probably worth removing one or the other of them :smile: